app/models/conference.rb

```diff
@@ -595,8 +595,38 @@ def registration_limit_exceeded?
     registration_limit > 0 && registrations.count >= registration_limit
   end
 
+  # Return a different html color for every new track. Consecutive talks have
+  # highly different colors.
+  def next_track_color
```

Comment:
Is it really necessary to have 3 different methods for the (almost) the same functionality?
I propose to either 

- just use next_color (I don't really care if a track and difficulty level have the same color)
- just use next_color and hardcode the start_index when you call the next_color

Three different methods are just too much for my taste ...